### DISCUSSION TITLE
PyCon Sprints - PyVista

### DISCUSSION THREAD

**Discussion by akaszynski** [pyvista/pyvista#2504] [Announcements]
PyVista will be a [PyCon Sprint](https://us.pycon.org/2022/events/development-sprints/#sprint-37)!

Depending on the number of participants, there may be a few PRs coming through as a result of the sprint. I'll keep everyone informed.

**Comment by tkoyama010** [comment]
@akaszynski Great! Are you going to attend the PyCon?

**Reply by akaszynski** [reply]
Yes, I'll be there. Not presenting this year, but if our [Tutorial](https://www.scipy2022.scipy.org/tutorials) goes well, I'll see if we can be added as a tutorial for next year.

**Reply by tkoyama010** [reply]
Great, I can help with Sprint during PyCon as it coincides with my extended vacation. I'm only online, but if you need support, give me a mention.

**Reply by JacobBumgarner** [reply]
I'll be at PyCon as well! I won't be there for the sprint, but I'll definitely try to find you to say hello at some point!

**Reply by tkoyama010** [reply]
Excellent, enjoy PyCon!
